gcp/pd-backed-nfs-server/manifests/pv-for-deployment-pods.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: PersistentVolume
 metadata:
   name: env0-nfs-pv
-  namespace: env0-agent
```

Comment:
PVs are global resources in k8s, hence they don't support namespace. Not sure why `kubectl` provider has tolerated it and `kubernetes` provider hasn't 🤷